script/setup-dev

```diff
@@ -1,57 +1,17 @@
 #!/usr/bin/env bash
 
-### Setup development environment
-###
-### Installs NodeJS and uses `corepack` to make `yarn` and `pnpm` available.
-###
-### Note: To update the version of NodeJS, update the `.node-version` file at
-### the root of the repository and add `hashes/node-v*.sha256` files for the
-### new version.
+# This used to install node and other dependencies.
+# That functionality has been migrated to Ansible.
+# This script now simply calls that functionality.
 
 set -euo pipefail
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
is there any harm in calling setup_ansible in here as well? I guess I'm wondering for VxDev where this script is automatically called, but they don't have ansible installed yet (we could also just make the update_code script in VxDev call install_ansible)

Reply:
No harm at all, great idea. I'll incorporate it with additional changes related to node/pnpm versioning.